internal/database/migrations/69_add_project_column_test.go

```
@@ -0,0 +1,290 @@
+
+		// Try to update the project column:
+		_, err = conn.Exec(ctx, `
+			update clusters set project = 'your-project' where id = '123'
+		`)
+		Expect(err).To(HaveOccurred())
+		Expect(err.Error()).To(ContainSubstring("immutable"))
+	})
+
+	It("Automatically creates the empty project for a tenant", func(ctx context.Context) {
+		// Run the migrations:
+		err := tool.Migrate(ctx, 50)
```

golangci-lint (2.12.2)

[error] ineffectual assignment to err

(ineffassign)